tests/cupyx_tests/scipy_tests/sparse_tests/test_construct.py

```diff
@@ -401,7 +401,7 @@ def test_diags_non_square(self, xp, sp):
     'arrA': arrs,
     'arrB': arrs,
 }))
-@testing.with_requires('scipy')
+@testing.with_requires('scipy>=1.6')
```

Comment:
Would this requirement be too strong? This method already exists before SciPy 1.6. Perhaps we can add the requirement at the method level?

Reply:
I think adding here it is fine, there is only one test and currently, it won't pass for scipy<1.6

Reply:
```python
if testing.scipy_satisfies('<1.6'):  # similar to testing.numpy_satisfies
    kron = kron.A
```

is an option.  But I justify the current fix as a kind of white-box test.  CuPy's `kron` does not directly refer `cupy.sparse`, i.e. the dependency `scipy>=1.6` is only for the testing purpose.